docs/platforms/javascript/common/configuration/integrations/vue.mdx

```diff
@@ -5,8 +5,37 @@ supported:
   - javascript.vue
 ---
 
+<Alert level="info">
+
+  This integration is enabled by default (recommended) but can also be added manually.
+
+</Alert>
+
 _Import name: `Sentry.vueIntegration`_
 
 This integration is enabled by default. If you'd like to modify your default integrations, read [this](./../#modifying-default-integrations).
 
 Adds error and span instrumentation for Vue applications.
+
+## Manual Setup
+
+If the Vue application is not defined from the start, you can add error monitoring for Vue-specific errors later on.
+To manually add the integration for late-defined Vue applications:
+
+```javascript
```

Comment:
I think it would help to either add a filename to the code sample (such as `main.js`, etc.)or specify where this code should go in the preceding sentence.